Source/FakeItEasy/Configuration/BuildableCallRule.cs

```diff
@@ -113,7 +113,7 @@ public void WriteDescriptionOfValidCall(IOutputWriter writer)
 
         public virtual void ApplyWherePredicate(Func<IFakeObjectCall, bool> predicate, Action<IOutputWriter> descriptionWriter)
         {
-            this.wherePredicates.Add(Tuple.Create(predicate, descriptionWriter));
+            this.wherePredicates.Add(new Tuple<Func<IFakeObjectCall, bool>, Action<IOutputWriter>>(predicate, descriptionWriter));

```

Comment:
I don't mind a force push. It's often better than keeping a series of commits which all have a single ultimate aim.